api/rest/stat_series.md

```diff
@@ -39,21 +37,23 @@ You can find a list of StatisticalVariables with human-readable names [here](/st
 
 ### Step 1: Assembling the information you will need
 
-This endpoint requires two arguments and offers four additional optional arguments, as listed:
+Going into more detail on how to assemble the values for the required arguments:
 
  - `place`: For this parameter, you will need to specify the DCID (the unique ID assigned by Data Commons to each node in the graph) of the place you are interested in.
  - `stat_var`: The statistical variable whose value you are interested in.
 
-In addition to these required properties, this endpoint also allows for other, optional arguments.
-
-  - `date`: If the property queried only takes on node values, you can use this argument to filter nodes in the response, ensuring the response only contains nodes with the specified type.
+In addition to these required properties, this endpoint also allows for other, optional arguments. Here are helpful arguments in regular use by Data Commons developers:
 
   - `measurement_method`: You can specify this argument as `out` to indicate that you desire the response to only include nodes which are supercategories of the specified `DCIDs`, or `in` to only return nodes that are subcategories of the specified `DCIDs`. (For example, South America is a supercategory of Argentina, which in turn is a supercategory of Buenos Aires, as illustrated in Figure 1.)
   
   - `observation_period`: (≤ 500) Maximum number of values returned per node.
```

Comment:
This description is also incorrect, and we should point to the glossary

Reply:
fixed :sob: